src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/LocalDB.Windows.cs

```diff
@@ -103,6 +233,9 @@ internal static string MapLocalDBErrorStateToErrorMessage(LocalDBErrorState erro
         /// </summary>
         private bool LoadUserInstanceDll()
         {
+            _sqlUserInstanceLibraryHandle = null;
+            throw new Exception();
```

Comment:
This was not meant to be part of the change. I may have been playing around with it on my AMD64 dev box, so I wanted to make it fail artificially. Oops...